src/client/delphi_epidata.py

```diff
@@ -102,6 +103,20 @@ def _request(endpoint, params={}):
         capabilities, but fall back to POST if the request is too
         long and returns a 414.
         """
+        # One-time check to verify the client version is up to date.
+        if Epidata.debug and not Epidata.sandbox and not Epidata.version_check:
+            try:
+                server_version = Epidata._request_with_retry("version").json()
+                if server_version["version"] != Epidata.client_version:
```

Comment:
Now that we've decoupled the server version from the client version, this check won't be very informative - the versions will often not match, but that won't mean that the client isn't up to date. We need to find another way to query the latest client version, I think.

Reply:
https://github.com/cmu-delphi/delphi-epidata/issues/1281#issuecomment-2104903462